<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lyricsSettings</class>
 <widget class="QWidget" name="lyricsSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>195</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lyrics Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="fontGroup">
     <property name="title">
      <string>Font</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="KFontRequester" name="fontChooser"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="KButtonGroup" name="alignmentGroup">
     <property name="title">
      <string>Alignment</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="alignLeft">
        <property name="text">
         <string comment="Left alignment">Left</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="alignCenter">
        <property name="text">
         <string comment="Center alignment">Center</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="alignRight">
        <property name="text">
         <string>Right</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KFontRequester</class>
   <extends>QWidget</extends>
   <header>kfontrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KButtonGroup</class>
   <extends>QGroupBox</extends>
   <header>kbuttongroup.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
